application/console/console.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require_relative '../../dependencies'
+
+class Console
+  include ConsoleHelper
+
+  def console_menu
+    output(I18n.t('Greetings'))
+    case user_input
+    when COMMANDS[:create] then create_account
+    when COMMANDS[:load]   then load_account
+    else                        run_exit
+    end
+    main_menu
+  end
+
+  def create_account
+    loop do
+      name = user_name
+      age = user_age
+      login = user_login
+      password = user_password
+      @account = Account.new(name: name, age: age, login: login, password: password)
+      @account.valid? ? break : output(@account.errors.join("\n"))
+    end
+    updating_db(@account)
+  end
+
+  def load_account
+    accounts_db = db_accounts
+    return create_the_first_account if accounts_db.empty?
+
+    loop do
+      @account = Account.find_account(log_and_pass, accounts_db)
+      @account.nil? ? output(I18n.t('ERROR.user_not_exists')) : break
+    end
+  end
+
+  def log_and_pass
```

Comment:
1. make it private
2. rename it to `login_and_password`

Reply:
made it private and renamed